src/process_user_code.py

```diff
@@ -43,7 +44,7 @@ def run(self):
             sys.stdin.flush()
             try:
                 new_state = json.loads(read_val)
-                for event in CONSTANTS.EXPECTED_INPUT_EVENTS:
+                for event in CONSTANTS.EXPECTED_INPUT_EVENTS_CPX:
```

Comment:
Why is this CPX only?

Reply:
yes, it's not in either library, so I'm assuming that it will be common